keyboards/cmm_studio/fuji65/readme.md

```diff
@@ -0,0 +1,17 @@
+# CMM.Studio Fuji65
+
+CMM.Studio Fuji65 Keyboard
+
+![CMM.Studio Fuji65](https://i.imgur.com/dUPQpkZ.png)
+
+* Keyboard Maintainer: [CMM.Studio Freather](https://github.com/CMMS-Freather)
+* Hardware Supported: PCB, Atmega32u4
+
+Make example for this keyboard (after setting up your build environment):

```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

Reply:
its a physical button to enter the bootloader mode. I wrote it under the make command

line 15